K2Bridge/Visitors/Aggregations/AggregationDictionaryVisitor.cs

```diff
@@ -67,10 +72,93 @@ public string BuildBucketAggregationQuery(BucketAggregation bucketAggregation, B
             query.Append($"{KustoQLOperators.CommandSeparator} {KustoQLOperators.Extend} {definition.ExtendExpression};");
 
             query.Append($"{KustoQLOperators.NewLine}{KustoQLOperators.Let} {AggregationsSubQueries.SummarizableMetricsQuery} = {AggregationsSubQueries.ExtDataQuery}");
-            query.Append($"{KustoQLOperators.CommandSeparator} {KustoQLOperators.Summarize} {bucketAggregation.SubAggregationsKustoQL}");
+            query.Append($"{KustoQLOperators.CommandSeparator} {KustoQLOperators.Summarize} {bucketAggregation.SummarizableMetricsKustoQL}");
             query.Append($"{definition.BucketExpression};");
 
+            query.Append($"{bucketAggregation.PartitionableMetricsKustoQL}");
+
             return query.ToString();
         }
+
+        public string BuildPartitionQuery(PartitionQueryDefinition definition)
```

Comment:
Why is this method (and the  two properties in the top of the file) in this visitor rather than the MetricsAggsVisitor?

Looks like its not used in this file, so it makes more sense to have in the other place

Reply:
Good point. Moved BuildPartitionQuery and BuildBucketAggregationQuery to helpers folder in dedicated files.